WoWPro_Leveling/Wrath/Horde/WOTLK_Dragonblight.lua

```diff
@@ -121,13 +121,13 @@ A Search Indu'le Village |QID|12106|M|38.1,46.2|PRE|12084|N|From the Image of Ar
 T A Letter for Home |QID|12085|M|38.1,46.3|N|To Overlord Agmar.|
 F Moa'ki Harbor|QID|11958|M|37.5,45.7|N|Fly to Moa'ki Harbor.|
 A Planning for the Future |QID|11960|N|From Trapper Mau'i.|M|48.30,74.30|
-h Moa'ki Harbor |QID|11960|N|Make this inn your home location.|
+h Moa'ki Harbor |ACTIVE|11960|N|Make this inn your home location.|M|48.15,74.76|
 A Let Nothing Go To Waste |QID|11958|N|From Elder Ko'nani.|M|48.10,74.90|
-l Snowfall Glade Pups |QID|11960|S|QO|1|N|Right-click 12 Snowfall Glade Pups, try not to do AoEs near them as it can kill the pups.|M|43.6,71.3|; Snowfall Glade Pup: 12/12
+C Snowfall Glade Pups |QID|11960|QO|1|N|Right-click 12 Snowfall Glade Pups, try not to do AoEs near them as it can kill the pups.|M|43.6,71.3|S|
```

Comment:
Is this a place to replace the steps using `S` and `US` with a single step with an `S!US` tag?

Reply:
I'm not entirely sure how the S!US functions so i am not sure.

Reply:
Yeah, a `S!US` tag substitutes for a uncomplicated pair of `S` and `US` tags in different steps.
The biggest difference is if they are immediately by a step that depends on it being done, in which case it might show up too early.

Reply:
This particular quest needs to be a S and a separate US because there are a few quests in the area that can be completed before the Pups are all collected.